docs/v0.4/getting-started/docker-for-mac.md

```diff
@@ -160,34 +160,101 @@ NAME     LATEST IMAGE
 square   index.docker.io/$DOCKER_ID/square@sha256:ac089ca183368aa831597f94a2dbb462a157ccf7bbe0f3868294e15a24308f68   square.js   <empty>   <empty>   Ready    1m13s
 ```
 
-## create a deployer
+## create a Knative deployer
```

Comment:
Can we link to the core/knative runtime docs instead?

Reply:
This is a tricky balance between using a one-page "getting started" to convey the concept of multiple runtimes which is one of the biggest new ideas for this release of riff, or erring on the side of shorter, just to get users to a place where something works.

I'm actually tempted to go in the other direction and show BOTH an app workload with the Core runtime, and a function workload with the Knative runtime.

cc @markfisher

Reply:
In the interest of shipping the docs, resolved to leave as-is for now, and do similar flow for the other 3 environments. 

One idea to consider for the future would be to have a single opinionated getting-started page wih just 1 environment and 1 runtime to create and deploy 1 function, and then also provide dedicated pages for installation in each environment, as well as pages for each workload type (functions, applications, containers).